core/src/main/java/org/akaza/openclinica/bean/extract/odm/ClinicalDataReportBean.java

```diff
@@ -595,5 +584,25 @@ public int compare(Object o1, Object o2) {
 	    });
 	}
 	
+	/**
+	 *  get emoji as plain text code
+	 * @param s
+	 * @return
+	 */
+	private String processEmoji(String s) {
```

Comment:
instead of processing the emoji, we may be able to fix it by upgrading xercesImpl. OC4 doesn't seem to have this issue

Reply:
after check the code , in OC4 the solution is to use comons-lang3, now change the OC3ocde to use the same solution